.changeset/funny-flies-collect.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+'@backstage/backend-app-api': patch
+'@backstage/backend-common': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/eslint-plugin': patch
+'@backstage/plugin-auth-backend': patch
+'@backstage/repo-tools': patch
+'@backstage/cli': patch
+---
+
+Bump dependency `minimatch` to v5
```

Comment:
`v9`?

Reply:
yep indeeed, got it mixed up with the other bump i made, thanks